### PR TITLE
Fixes not properly parsing certificates in Java Adapter library

### PR DESCRIPTION
**Pull request by ahmed-sadak**
Closes #365

### CODE REVIEW

File: lib/java/AdapterLibrary/build.gradle.kts [hunk 1]
```diff
@@ -98,6 +98,6 @@ publishing {
     }
 }
 
-signing {
-    sign(publishing.publications["mavenJava"])
-}
+//signing {
+//    sign(publishing.publications["mavenJava"])
+//}
```
Comment (kjrokos) [inline lib/java/AdapterLibrary/build.gradle.kts:103]: We'll need this to publish to Maven. It shouldn't prevent you from building locally - but either way, shouldn't be committed.

File: lib/java/AdapterLibrary/src/test/java/com/vmware/aria/operations/AdapterInstanceTest.java [hunk 2]
```diff
@@ -116,6 +101,30 @@ private JsonPrimitive nil() {
...
+    }
+
+    private JsonPrimitive n(Number i) {
+        return JsonElementKt.JsonPrimitive(i);
+    }
+
+    private JsonPrimitive nil() {
+        return JsonElementKt.JsonPrimitive((Boolean) null);
+    }
+
```
Comment (kjrokos) [inline lib/java/AdapterLibrary/src/test/java/com/vmware/aria/operations/AdapterInstanceTest.java:127]: I'm curious why you moved these

**Review by kjrokos — CHANGES_REQUESTED** [review]
The lint check that's failing isn't related to your code. It's probably better fixed in a separate PR, so you can ignore for now.

File: lib/java/AdapterLibrary/build.gradle.kts (hunk 1 above)
Comment (ahmed-sadak) [reply lib/java/AdapterLibrary/build.gradle.kts]: my bad, forgot to revert this after testing locally

File: lib/java/AdapterLibrary/src/test/java/com/vmware/aria/operations/AdapterInstanceTest.java (hunk 2 above)
Comment (ahmed-sadak) [reply lib/java/AdapterLibrary/src/test/java/com/vmware/aria/operations/AdapterInstanceTest.java:127]: these were moved by my default formatter

Comment (kjrokos) [reply lib/java/AdapterLibrary/src/test/java/com/vmware/aria/operations/AdapterInstanceTest.java:127]: Ah, that makes sense. Is that just the default IntelliJ settings?

Comment (ahmed-sadak) [reply lib/java/AdapterLibrary/src/test/java/com/vmware/aria/operations/AdapterInstanceTest.java:127]: Yes, if there are any specific ones for this project I can import them, reformat and resubmit

**Review by kjrokos — APPROVED** [review]

**Pull request merged**